source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/control/impedance_scene_cfg.py

```diff
@@ -0,0 +1,151 @@
+                    prim_path=MISSING,
+                    name="end_effector",
+                    offset=OffsetCfg(
+                        pos=(0, 0, 0),
+                    ),
+                ),
+            ],
+        )
+
+    ground = AssetBaseCfg(
+        prim_path="/World/defaultGroundPlane",
+        spawn=sim_utils.GroundPlaneCfg(),
+    )
+
+    dome_light = AssetBaseCfg(
+        prim_path="/World/Light", spawn=sim_utils.DomeLightCfg(intensity=3000.0, color=(0.75, 0.75, 0.75))
+    )
+
+    table = RigidObjectCfg(
+        prim_path="{ENV_REGEX_NS}/Table",
+        spawn=sim_utils.CuboidCfg(
+            size=(1.25, 1.0, 0.3),
+            collision_props=sim_utils.CollisionPropertiesCfg(),
+            visual_material=sim_utils.PreviewSurfaceCfg(),
+            rigid_props=sim_utils.RigidBodyPropertiesCfg(kinematic_enabled=True, max_depenetration_velocity=0.1),
+            activate_contact_sensors=True,
+        ),
+        init_state=RigidObjectCfg.InitialStateCfg(
+            pos=(0.8, 0.0, 0.15), rot=(1.0, 0.0, 0.0, 0.0)
+        ),
+    )
+
+    contact_sensor = ContactSensorCfg(
+        prim_path="{ENV_REGEX_NS}/Robot/wrist_3_link",
+        update_period=0.0,
+        history_length=2,
+        debug_vis=True,
+        filter_prim_paths_expr=["{ENV_REGEX_NS}/Table"],
+    )
+
+##
+# MDP settings
+##
+
+
+@configclass
+class CommandsCfg:
+    """Command terms for the MDP."""
+    pass
+
+
+@configclass
+class ActionsCfg:
+    """Action specifications for the MDP."""
+
+    arm_action: ActionTerm = MISSING
+
+
+@configclass
+class ObservationsCfg:
+    """Observation specifications for the MDP."""
+
+    @configclass
+    class PolicyCfg(ObsGroup):
+        """Observations for policy group."""
+        pass
+    # observation groups
+    policy: PolicyCfg = PolicyCfg()
+
+
+@configclass
+class EventCfg:
+    """Configuration for events."""
+    pass
+
+
+@configclass
+class RewardsCfg:
+    """Reward terms for the MDP."""
+    pass
+
+@configclass
+class TerminationsCfg:
+    """Termination terms for the MDP."""
+    pass
+
+@configclass
+class CurriculumCfg:
+    """Curriculum terms for the MDP."""
+    pass
+
+
+##
+# Environment configuration
+##
+
+
+@configclass
+class ImpedanceControlRLSceneCfg(ManagerBasedRLEnvCfg):
+    """Configuration for the lifting environment."""
+
+    # Scene settings
+    scene: ImpedanceControlSceneCfg = ImpedanceControlSceneCfg(num_envs=4096, env_spacing=2.5)
+    # Basic settings
+    observations: ObservationsCfg = ObservationsCfg()
+    actions: ActionsCfg = ActionsCfg()
+    commands: CommandsCfg = CommandsCfg()
+    # MDP settings
+    rewards: RewardsCfg = RewardsCfg()
+    terminations: TerminationsCfg = TerminationsCfg()
+    events: EventCfg = EventCfg()
+    curriculum: CurriculumCfg = CurriculumCfg()
+
+    def __post_init__(self):
+        """Post initialization."""
+        # general settings
+        self.decimation = 2
+        self.sim.render_interval = self.decimation
```

Comment:
how are those 2 correlated?


```suggestion
        self.decimation = 2
        self.sim.render_interval = 2
```